atd-etl/app/process/helpers_socrata.py

```diff
@@ -311,3 +312,16 @@ def format_person_data(data, formatter_config):
     formatted_records = concatTimeAndDate(formatted_records)
 
     return formatted_records
+
+
+def get_date_limit():
+    """
+    Returns a string containing today's date 5 years ago in iso format: yyyy-mm-dd
```

Comment:
I think that this comment may need to be updated to describe the returned value (1st day of the previous month). I like this though as I have been using the day before this (last day of the month that is 2 months ago). Way easier to put this into words. 🎉

Reply:
Ah, good call. Let me update it. 🙏